opm/input/eclipse/EclipseState/SimulationConfig/BCConfig.hpp

```diff
@@ -33,36 +33,15 @@ namespace Opm {
 class Deck;
 class DeckRecord;
 
-enum class BCType {
-     RATE,
-     FREE,
-     DIRICHLET,
-     THERMAL
-};
-
-enum class BCComponent {
-     OIL,
-     GAS,
-     WATER,
-     SOLVENT,
-     POLYMER,
-     NONE
-};
-
-
 class BCConfig {
 public:
 
     struct BCFace {
```

Comment:
It is confusing to have the same name for the nested BCFace class in two different classes. Maybe replace this one with BCRegion? It is after all not a single face usually. The other can be BCSpec or BCProp (singular, as opposed to BCProps) or kept at BCFace if you prefer.